modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -13,105 +13,33 @@ Required: Yes +
 |Describe any important details for this endpoint. +
 Required: No +
 
-| AS2 URL (Automatically generated at time of deployment)
+| AS2 URL
 | The HTTP or HTTPS address where you receive AS2 messages.
-The port is automatically generated when you deploy a message flow that uses this endpoint.
-You can see it by selecting the receiving endpoint from your deployed message flow.
-The hostname displayed is created by your administrator at setup.
 
+When deploying to CloudHub with a public Shared Load Balancer, the URL is automatically generated when you deploy a message flow that uses this endpoint.
 
-| Message Signature Encryption
-| If your partner sends an encrypted signature in the message, choose the appropriate algorithm for validating that signature: +
+To get the external facing URL of the AS2 receive endpoint deployed from Partner Manager (when deploying to CloudHub with Dedicated load balancer) refer to https://docs.mulesoft.com/runtime-manager/cloudhub-dedicated-load-balancer
```

Comment:
```suggestion
To get the external-facing URL of the AS2 receive endpoint deployed from Partner Manager (when deploying to CloudHub by using a dedicated load balancer), refer to https://docs.mulesoft.com/runtime-manager/cloudhub-dedicated-load-balancer.
```
...but do we really want to introduce a goal and then send the audience off to some other source to tell how to achieve that goal?

Reply:
It's a set of steps only related to load balancer so yes, I think so.